dockerfile-mode.el

```diff
@@ -203,8 +205,18 @@ This can be set in file or directory-local variables.")
   "Build an image called IMAGE-NAME based upon the buffer.
 
 If prefix arg NO-CACHE is set, don't cache the image.
+
 The build string will be of the format:
-`sudo docker build --no-cache --force-rm --pull --tag IMAGE-NAME --build-args arg1.. --progress PROGRESS_TYPE -f filename directory`"
+
+    sudo docker build
```

Comment:
If this is supposed to look like a shell command, there should be backslashes at the end the the "not last" lines.

Reply:
OK. Fixed.